bioimageio_core_version: 0.5.9
bioimageio_spec_version: 0.4.9
rdf_sha256: e2ab2b205715ec505ea4fce90af20e91e139ee3295de8f1d1d5a6f0a52a9e56b
status: failed
tests:
  bioimageio:
  - bioimageio_spec_version: 0.4.9
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.9 with update to latest
      format version
    nested_errors: {}
    status: passed
    traceback: null
    warnings: {tags: 'Missing tags corresponding to bioimage.io categories: [{''modality'':
        [''electron-microscopy'', ''cryo-electron-microscopy'', ''fluorescence-light-microscopy'',
        ''transmission-light-microscopy'', ''super-resolution-microscopy'', ''x-ray-microscopy'',
        ''force-microscopy'', ''high-content-imaging'', ''whole-slide-imaging'', ''other'']},
        {''dims'': [''2d'', ''3d'', ''2d-t'', ''3d-t'']}, {''framework'': [''tensorflow'',
        ''pytorch'', ''tensorflow.js'']}, {''network'': [''unet'', ''densenet'', ''resnet'',
        ''inception'', ''shufflenet'']}, {''task'': [''semantic-segmentation'', ''instance-segmentation'',
        ''object-detection'', ''image-classification'', ''denoising'', ''image-restoration'',
        ''image-reconstruction'', ''in-silico-labeling'']}]'}
  - bioimageio_spec_version: 0.4.9
    error: null
    name: bioimageio.spec static validation of model RDF 0.4.9
    nested_errors: {}
    status: passed
    traceback: null
    warnings: {tags: 'Missing tags corresponding to bioimage.io categories: [{''modality'':
        [''electron-microscopy'', ''cryo-electron-microscopy'', ''fluorescence-light-microscopy'',
        ''transmission-light-microscopy'', ''super-resolution-microscopy'', ''x-ray-microscopy'',
        ''force-microscopy'', ''high-content-imaging'', ''whole-slide-imaging'', ''other'']},
        {''dims'': [''2d'', ''3d'', ''2d-t'', ''3d-t'']}, {''framework'': [''tensorflow'',
        ''pytorch'', ''tensorflow.js'']}, {''network'': [''unet'', ''densenet'', ''resnet'',
        ''inception'', ''shufflenet'']}, {''task'': [''semantic-segmentation'', ''instance-segmentation'',
        ''object-detection'', ''image-classification'', ''denoising'', ''image-restoration'',
        ''image-reconstruction'', ''in-silico-labeling'']}]'}
  - bioimageio_core_version: 0.5.9
    bioimageio_spec_version: 0.4.9
    error: 'Failed to download https://zenodo.org/api/files/e4c6af78-f693-4214-844a-05b08ef5fa27/cover.png
      (500 Server Error: INTERNAL SERVER ERROR for url: https://zenodo.org/api/files/e4c6af78-f693-4214-844a-05b08ef5fa27/cover.png)'
    name: load resource description
    nested_errors: null
    status: failed
    traceback: ["  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/core/resource_tests.py\",
        line 201, in _test_load_resource\n    rd: Optional[ResourceDescription] =
        load_resource_description(\n", "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/core/resource_io/io_.py\",
        line 57, in load_resource_description\n    rd: ResourceDescription = resolve_raw_node(raw_rd=raw_rd,
        nodes_module=nodes)\n", "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/core/resource_io/utils.py\",
        line 121, in resolve_raw_node\n    rd = UriNodeTransformer(root_path=raw_rd.root_path,
        uri_only_if_in_package=uri_only_if_in_package).transform(raw_rd)\n", "  File
        \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/spec/shared/node_transformer.py\",
        line 56, in transform\n    return transformer(node, **kwargs)  # noqa\n",
      "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/spec/shared/node_transformer.py\",
        line 161, in generic_transformer\n    node, **{n: self.transform(value, name=n,
        parent=node) for n, value in iter_fields(node)}\n", "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/spec/shared/node_transformer.py\",
        line 161, in <dictcomp>\n    node, **{n: self.transform(value, name=n, parent=node)
        for n, value in iter_fields(node)}\n", "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/spec/shared/node_transformer.py\",
        line 56, in transform\n    return transformer(node, **kwargs)  # noqa\n",
      "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/spec/shared/node_transformer.py\",
        line 62, in transform_list\n    return [self.transform(subnode, **kwargs)
        for subnode in node]\n", "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/spec/shared/node_transformer.py\",
        line 62, in <listcomp>\n    return [self.transform(subnode, **kwargs) for
        subnode in node]\n", "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/spec/shared/node_transformer.py\",
        line 56, in transform\n    return transformer(node, **kwargs)  # noqa\n",
      "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/spec/shared/node_transformer.py\",
        line 339, in transform_URI\n    local_path = _resolve_source(node, root_path=self.root)\n",
      "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/functools.py\",
        line 889, in wrapper\n    return dispatch(args[0].__class__)(*args, **kw)\n",
      "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/spec/shared/_resolve_source.py\",
        line 253, in _resolve_source_uri_node\n    local_path = _download_url(path_or_remote_uri,
        output, pbar=pbar)\n", "  File \"/home/runner/micromamba-root/envs/7786493/lib/python3.10/site-packages/bioimageio/spec/shared/_resolve_source.py\",
        line 479, in _download_url\n    raise RuntimeError(f\"Failed to download {uri}
        ({e})\") from e\n"]
    warnings: {}
